### DISCUSSION TITLE
Modifing mod

### DISCUSSION THREAD

**Discussion by C3LT1C0** [ramensoftware/windhawk-mods#1184] [Q&A]
While browsing, I found these scripts for the taskbar, start menu, and notification center mods.
I like the look they provide, but they're not perfect.
Can I ask if anyone knows how to modify them and make everything uniform? Specifically, the right part of the start menu should have the same degree of transparency, as well as the hidden icons on the taskbar.
I have included screenshots and the files with the scripts.
Thanks for the help.

![Screenshot 2024-11-01 120005](https://github.com/user-attachments/assets/9307660b-61ec-4e90-95c0-e65b37b2ef53)
![Screenshot 2024-11-01 120027](https://github.com/user-attachments/assets/bfd3ecc5-8983-43d6-8555-57735c6337cb)
![Screenshot 2024-11-01 120115](https://github.com/user-attachments/assets/48e17658-f163-4a75-9288-2402b4d54ae9)
[Notification Center Codes.txt](https://github.com/user-attachments/files/17598929/Notification.Center.Codes.txt)
[Start Menu Styler Codes.txt](https://github.com/user-attachments/files/17598930/Start.Menu.Styler.Codes.txt)
[Taskbar Styler Codes.txt](https://github.com/user-attachments/files/17598931/Taskbar.Styler.Codes.txt)

**Comment by m417z** [comment]
A better place to ask about it would be here:
https://github.com/ramensoftware/windows-11-start-menu-styling-guide/issues

You can also try [the Windhawk Discord server](https://discord.com/servers/windhawk-923944342991818753), we have a channel that's dedicated for styling.
